src/domain/common/utils/signatures.ts

```diff
@@ -0,0 +1,85 @@
+const HEX_PREFIX_LENGTH = 2;
+export const SIGNATURE_HEX_LENGTH = 65 * 2; // 65 bytes in hex
+export const DYNAMIC_PART_LENGTH_FIELD_HEX_LENGTH = 64; // 32 bytes in hex
+
+/**
+ * Parses a (concatenated) signature string into inidividual signature types
```

Comment:
Nit: `...into individual signature types`

Reply:
Fixed in 74332901f1a21437d37173f2813dc4c2fb1e3433.